app/models/user.rb

```diff
@@ -0,0 +1,3 @@
+class User < ApplicationRecord
+  has_many :tasks
```

Comment:
The same comment here.
```suggestion

  validates :name, :email, presence: true, uniqueness: true
```